apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagRoutes.kt

```diff
@@ -40,13 +40,14 @@ fun Route.grunnlagRoute(grunnlagService: GrunnlagService) {
         }
 
         get("{sakId}/{opplysningType}") {
-            val grunnlag = grunnlagService.hentGrunnlagAvType(
-                call.parameters["sakId"]!!.toLong(),
-                Opplysningstype.valueOf(call.parameters["opplysningType"].toString())
-            )
+            val opplysningstype = Opplysningstype.valueOf(call.parameters["opplysningType"].toString())
+            val sakId = call.parameters["sakId"]!!.toLong()
+            val grunnlag = grunnlagService.hentGrunnlagAvType(sakId, opplysningstype)
 
             if (grunnlag != null) {
                 call.respond(grunnlag)
+            } else if (opplysningstype == Opplysningstype.SOESKEN_I_BEREGNINGEN) {
+                call.respond(HttpStatusCode.NoContent)
```

Comment:
Vil dette fungere med mappingen i frontend? Eller klarer den å kjøre `.text` (eller blir det `.json`?) på en tom response-body?

Reply:
Det fungerte i front da jeg testet det! Kan installere branchene en gang til for å dobbeltsjekke

Reply:
Du har rett Øyvind! Den får en type-error i front, men funksjonaliteten fungerer fint. Det blir litt type-problematikk med responsen ettersom vi sier det er en success, og forventer dermed å få type `U` mens 204 gir en tom streng. Mulig dette skal legges på en egen jira-task

Reply:
Mekka en commit som gjør at det ikke kommer noen feil på dette i console